src/main/java/uk/gov/companieshouse/overseasentitiesapi/service/ManagingOfficerChangeService.java

```diff
@@ -0,0 +1,310 @@
+        changes.addAll(getIndividualManagingOfficerChange());
+        changes.addAll(getCorporateManagingOfficerChange());
+        return changes;
+    }
+
+
+    /**
+     * Identifies and returns changes related to the 'Individual' type managing officers in the
+     * submission.
+     *
+     * <p>This method is responsible for tracking the changes made to the managing officers of
+     * 'Individual' type within the provided submission. Each change is encapsulated within a
+     * {@code ManagingOfficerChange<IndividualManagingOfficer>} object.</p>
+     *
+     * @return A list of changes ({@code ManagingOfficerChange<IndividualManagingOfficer>} objects)
+     * related to the managing officers of 'Individual' type. If no changes are detected, an empty
+     * list is returned.
+     */
+    private List<Change> getIndividualManagingOfficerChange() {
+        var individualManagingOfficers = overseasEntitySubmissionDto.getManagingOfficersIndividual();
+        return individualManagingOfficers
+                .stream()
+                .filter(mo -> mo.getChipsReference() != null)
+                .map(this::covertManagingOfficerIndividualToChange)
+                .filter(Objects::nonNull)
+                .collect(Collectors.toList());
+    }
+
+
+    /**
+     * Identifies and returns changes related to the 'Corporate' type managing officers in the
+     * submission.
+     *
+     * <p>This method is responsible for tracking the changes made to the managing officers of
+     * 'Corporate' type within the provided submission. Each change is encapsulated within a
+     * {@code ManagingOfficerChange<CorporateManagingOfficer>} object.</p>
+     *
+     * @return A list of changes ({@code ManagingOfficerChange<CorporateManagingOfficer>} objects)
+     * related to the managing officers of 'Corporate' type. If no changes are detected, an empty
+     * list is returned.
+     */
+    private List<Change> getCorporateManagingOfficerChange() {
+        var corporateManagingOfficers = overseasEntitySubmissionDto.getManagingOfficersCorporate();
+        return corporateManagingOfficers
+                .stream()
+                .filter(mo -> mo.getChipsReference() != null)
+                .map(this::covertManagingOfficerCorporateToChange)
+                .filter(Objects::nonNull)
+                .collect(Collectors.toList());
+    }
+
+    /**
+     * Converts a ManagingOfficerIndividualDto into a ManagingOfficerChange object.
+     *
+     * <p>The method detects changes between the provided managingOfficerIndividualDto and its
+     * corresponding information in the public and private data.</p>
+     *
+     * @param managingOfficerIndividualDto The ManagingOfficerIndividualDto object to be converted.
+     * @return A ManagingOfficerChange object if changes are detected, otherwise null.
+     */
+    private ManagingOfficerChange<IndividualManagingOfficer> covertManagingOfficerIndividualToChange(
```

Comment:
These should all be co**N**vert. Quick search through the file and add the Ns :-)